File: checkers/cwe/tests/suite/NullPtrDereferenceChecker/java/array_items_assign_in_function.java
class A {
    void func(String[] fo111, String[] fo222) {
        fo111[1] = fo222[0];
    }

    int main() {
        String[] a = new String[10];
        String[] b = new String[5];
        b[0] = null;
        func(a, b);
        return a[1].length();
    }
}

File: checkers/cwe/tests/suite/NullPtrDereferenceChecker/array_items_assign_in_function.cpp
void func(int* fo111[10], int* fo222[5]) {
    fo111[1] = fo222[0];
}

int main() {
    int* a[10];
    int* b[5];
    b[0] = 0;
    func(a, b);
    return *a[1];
}
